docs/dev/protocols.md

```diff
@@ -149,46 +149,35 @@ The CQL protocol support can be disabled altogether by setting the
 
 These option names were chosen for backward-compatibility with Cassandra
 configuration files: they refer to CQL as the "native transport", to
-contrast with the older Thrift protocol (described below) which wasn't
-native to Cassandra.
+contrast with the older Thrift protocol which wasn't
+native to Cassandra. The thrift protocol was once supported by Scylla,
+but the support of this protocol was later deprecated and removed.
 
 There is also a `rpc_address` configuration option to set the IP address
 (and therefore network interface) on which Scylla should listen for the
 CQL protocol. This address defaults to `localhost`, but in any setup except
-a one-node test, should be overridden. Note that the same option `rpc_address`
-applies to both CQL and Thrift protocols.
-
-TODO: there is also `rpc_interface` option... Which wins? What's the default?
-
-## Thrift client protocol
+a one-node test, should be overridden.
 
 The Apache Thrift protocol was early Cassandra's client protocol, until
 it was superseded in Cassandra 1.2 with the binary CQL protocol. Thrift
 was still nominally supported by both Cassandra and Scylla for many years,
-but was recently dropped in Cassandra (version 4.0) and is likely to be
-dropped by Scylla in the future as well, so it is not recommended for new
-applications.
+but was recently dropped in Cassandra (version 4.0) and was also
+dropped by Scylla in version 5.5 as well.
```

Comment:
Isn't this 6.0?

Reply:
yes. this document was updated before we cut 6.0. updated accordingly.